### ISSUE TITLE
alembic fails to build on 10.10.3

### ISSUE THREAD

**Issue by bfontaine** [Homebrew/homebrew-science#2536]
```
==> Installing homebrew/science/alembic
==> Cloning https://code.google.com/p/alembic/
Updating /Library/Caches/Homebrew/alembic--hg
==> Checking out tag 1_05_04
==> cmake . -DCMAKE_C_FLAGS_RELEASE= -DCMAKE_CXX_FLAGS_RELEASE= -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/alembic/1.5.4 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -
-- Skipping Arnold plugins
-- Skipping Maya plugins
-- Skipping Alembic Python bindings
-- Configuring incomplete, errors occurred!
See also "/tmp/alembic20150722-64576-2htwvt/CMakeFiles/CMakeOutput.log".
```

Logs: https://gist.github.com/1678f2b81d8020adc7f4

**Comment by dpo** [comment]
Does it build if you `brew install boost-python`?

**Comment by iml** [comment]
Fixed in #2590.
